apps/web/pages/api/user/me.ts

```diff
@@ -1,42 +0,0 @@
-import type { NextApiRequest, NextApiResponse } from "next";
-
-import { deleteStripeCustomer } from "@calcom/app-store/stripepayment/lib/customer";
-import prisma from "@calcom/prisma";
-
-import { getSession } from "@lib/auth";
-
-export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
Are we sure this is not being used anymore?

Reply:
I couldn't find anywhere else where it was being called besides the delete account feature, so I'm pretty sure yeah